docs/plugins/mapper-size.asciidoc

```diff
@@ -26,7 +26,7 @@ PUT my-index-000001
 --------------------------
 
 The value of the `_size` field is accessible in queries, aggregations, scripts,
-and when sorting:
+and when sorting. It can be retrieved using the {ref}/search-fields.html#search-fields-param[fields API]:
```

Comment:
I just noticed this: we've been trying not to call it the 'fields API' in official docs. It has caused confusion in the past because some users thought it was a separate API endpoint. We could say `fields` parameter or `fields` option instead.

Reply:
Oops, not sure this was updated before merging?

Reply:
Indeed, I missed that one, only changed the mention below in the example. Will push a small online-fix.

Reply:
Corrected this.